conans/__init__.py

```diff
@@ -10,6 +12,64 @@
 from conans.util.files import load
 
 
+class ProfileOptions(defaultdict):
```

Comment:
These classes cannot be here, this is too global of scope. Better move them to ``conan.profile``